watsononlinestore/watson_online_store.py

```diff
@@ -84,12 +84,17 @@ def __init__(self, bot_id, slack_client,
         # IBM Cloudant noSQL database
         self.cloudant_online_store = cloudant_online_store
 
+        self.at_bot = "<@" + bot_id + ">"
+        self.delay = 0.5  # second
+        self.workspace_id = os.environ.get("WORKSPACE_ID")
```

Comment:
this change is unrelated. These lines were moved around recently so probably a merge issue.  Also the workspace line will change soon with my PR 🤞